### PR TITLE
Fix or improve labels

### LINKED ISSUE #276: Map/curate 344 EFO measurement branch terms to OBA traits
- [x] Map EFO measurement branch terms to OBA traits
- [x] Create new OBA trait terms as needed

Related to
https://github.com/obophenotype/bio-attribute-ontology/issues/162#issuecomment-1658920829

### PR DESCRIPTION
**Pull request by rays22**
If applied, this commit will fix #276 .

### CODE REVIEW

File: src/patterns/data/default/entity_attribute.tsv [hunk 1]
```diff
@@ -4313,11 +4313,11 @@ OBA:2045266	carotid artery wall area attribute	UBERON:0037094	wall of common car
...
 OBA:2045268	striated muscle fiber area attribute	CL:0000737	striated muscle cell	PATO:0001323	area			https://orcid.org/0000-0001-8314-2140
 OBA:2045269	aortic vascular smooth muscle cell calcification attribute	CL:0002539	aortic smooth muscle cell	PATO:0001447	calcified			https://orcid.org/0000-0001-8314-2140
-OBA:2045270	basal ganglia growth attribute	UBERON:0002420	basal ganglion	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045271	brain growth attribute	UBERON:0000955	brain	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045272	cerebellum growth attribute	UBERON:0002037	cerebellum	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045273	cerebral cortex growth attribute	UBERON:0000956	cerebral cortex	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045274	prefrontal cortex growth attribute	UBERON:0000451	prefrontal cortex	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
+OBA:2045270	basal ganglia growth characteristic	UBERON:0002420	basal ganglion	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
```
Comment (ar-ibrahim) [inline src/patterns/data/default/entity_attribute.tsv:4316]: @rays22 Why were 'attribute' terms replaced by 'characteristic'?

Comment (rays22) [reply src/patterns/data/default/entity_attribute.tsv:4316]: It sound better. It was suggested by James at the meeting last Friday.

**Review by matentzn — COMMENTED** [review]
LGTM but I would ask in OBO channel what people think of the changing "X attribute" labels to "X characteristic" labels. This gives @cmungall and @dosumis a chance to but in (this is not a minor change)

**Comment by rays22** [comment]
> LGTM but I would ask in OBO channel what people think of the changing "X attribute" labels to "X characteristic" labels. This gives @cmungall and @dosumis a chance to but in (this is not a minor change)

I am open to suggestions and input on the labels, but I am not sure if this label change is a major change. I would think that `attribute` and `characteristic` [are synonyms.](https://www.thesaurus.com/browse/attribute):
* *characteristic*: [a typical or noticeable quality of someone or something](https://dictionary.cambridge.org/dictionary/english/characteristic)
* *attribute*: [a quality or characteristic that someone or something has](https://dictionary.cambridge.org/dictionary/english/attribute)

**Comment by rays22** [comment]
I have reverted the `characteristic` in the labels to `attribute`. We can come back and change the labels in another PR, but now I want to merge this one, because it fixes some typos for a new OBA release.

**Pull request merged**